src/Symfony/Component/EventDispatcher/DependencyInjection/RegisterListenersPass.php

```diff
@@ -108,7 +105,26 @@ public function process(ContainerBuilder $container)
                 throw new InvalidArgumentException(sprintf('Service "%s" must implement interface "%s".', $id, $interface));
             }
 
-            $definition->addMethodCall('addSubscriberService', array($id, $class));
+            $r = new \ReflectionClass($class);
+            $extractingDispatcher->addSubscriber($r->newInstanceWithoutConstructor());
```

Comment:
Can we call static method without instanciation https://github.com/symfony/symfony/blob/master/src/Symfony/Component/EventDispatcher/EventSubscriberInterface.php#L45 ?